app/views/state_file/questions/data_transfer_offboarding/edit.html.erb

```diff
@@ -4,7 +4,7 @@
 <% content_for :card do %>
   <h1 class="h2"><%= title %></h1>
 
-  <p><%= t('.offboarding_reason', reason: ineligible_reason) %></p>
+  <p><%= t('.offboarding_reason', reason: ineligible_reason).html_safe %></p>
```

Comment:
does this error somehow without the `html_safe`?

Reply:
It does not split into two lines without html_safe